lta/picker.py

```diff
@@ -110,16 +116,18 @@ def __init__(self, config: Dict[str, str], logger: Logger) -> None:
     # attempt to PATCH the status resource
     picker.logger.info(f"PATCH {status_url} - {status_body}")
     try:
-        session = FuturesSession()
-        # r = requests.patch(status_url, data=status_body)
-        r = await asyncio.wrap_future(session.patch(status_url, data=status_body))
-        if (r.status_code < 200) or (r.status_code > 299):
-            picker.logger.error("Unable to PATCH /status/picker with heartbeat")
-            picker.lta_ok = False
-            return picker.lta_ok
+        # TODO: Will probably refactor this into some function -- get a configured RestClient
+        rc = RestClient(picker.lta_rest_url,
+                        token=picker.lta_rest_token,
+                        timeout=picker.heartbeat_patch_timeout_seconds,
+                        retries=picker.heartbeat_patch_retries)
+        # Use the RestClient to PATCH our heartbeat to the LTA REST DB
+        await rc.request('PATCH', "/status/picker", status_body)
         picker.lta_ok = True
-    except requests.exceptions.ConnectionError:
-        picker.logger.error("ConnectionError trying to PATCH /status/picker with heartbeat")
+    except HTTPError as e:
+        # if there was a problem, yo I'll solve it
+        picker.logger.error("Error trying to PATCH /status/picker with heartbeat")
+        picker.logger.error(f"Error was: '{e}'")
         picker.lta_ok = False
```

Comment:
What if some other exception occurs?  We should still catch those, so we can continue heartbeating.

Reply:
Changed this to `Exception` so we can catch whatever goes wrong.